packages/zowe-explorer/src/dataset/ZoweDatasetNode.ts

```diff
@@ -131,6 +131,46 @@ export class ZoweDatasetNode extends ZoweTreeNode implements IZoweDatasetTreeNod
         }
     }
 
+    /**
+     * Updates an existing data set node that was recalled so it can be interacted with.
+     * @param isPds Whether the data set is a PDS
+     */
+    private datasetRecalled(isPds: boolean): void {
+        // Change context value to match dsorg, update collapsible state
+        this.contextValue = isPds ? globals.DS_PDS_CONTEXT : globals.DS_DS_CONTEXT;
```

Comment:
What if it's favorited?

Reply:
Good call - it looks like in the logic before this PR, we assumed it wasn't a favorite and set the context value based on collapsible state. I addressed this in 1ecdb88.